WorkflowUI/Sources/ViewControllerDescription/ViewControllerDescription.swift

```diff
@@ -126,8 +140,43 @@ public struct ViewControllerDescription {
             """
         )
 
+        configureAncestor(for: viewController)
+
         update(viewController)
     }
+
+    private func configureAncestor(for viewController: UIViewController) {
+        guard let ancestorOverride = viewController.environmentAncestorOverride else {
+            // If no ancestor is currently present establish the initial ancestor override
+            establishAncestorOverride(for: viewController)
+            return
+        }
+
+        let currentAncestor = ancestorOverride()
+        // Check whether the VC's ancestor was overridden by a ViewControllerDescription.
+        guard currentAncestor is PropagationNode else {
+            // Do not override the VC's ancestor if it was overridden by something outside of the
+            // `ViewControllerDescription`'s management of this node.
+            // The view controller we're managing, or the container it's contained in, likely needs to manage this in a
+            // special way.
+            return
+        }
+
+        // We must nil this out first or we'll hit an assertion which protects against overriding the ancestor when
+        // some other system has already attempted to provide an override.
+        viewController.environmentAncestorOverride = nil
+        establishAncestorOverride(for: viewController)
+    }
+
+    private func establishAncestorOverride(for viewController: UIViewController) {
+        let ancestor = PropagationNode(
+            viewController: viewController,
+            environment: environment
+        )
+        viewController.environmentAncestorOverride = { ancestor }
+
+        ancestor.setNeedsEnvironmentUpdate()
```

Comment:
```suggestion
        viewController.setNeedsEnvironmentUpdate()
```
same?

Reply:
I probably would, only because it slightly better reflects the intent. Don't mind either way.

Reply:
e437c72